d2bs/kolbot/libs/common/Pickit.js

```diff
@@ -245,7 +245,7 @@ const Pickit = {
 			this.color = Pickit.itemColor(unit);
 			this.gold = unit.getStat(14);
 			this.dist = (unit.distance || Infinity);
-			this.useTk = (Skill.haveTK && Config.UseTelekinesis
+			this.useTk = (Skill.haveTK
```

Comment:
should retain a way to disable TK

Reply:
This does still have a way to disable it, Config.Telekinesis is handled in the Skill.skills object